crmsh/command.py

```diff
@@ -231,8 +231,11 @@ def prev_completions():
 
 
 def _help_completer(args, context):
-    'TODO: make better completion'
-    return help_module.list_help_topics() + context.current_level().get_completions()
+    current_level_completions = context.current_level().get_completions()
+    help_topics_completions = help_module.list_help_topics()
+    if context.current_level().name == 'root':
+        return help_topics_completions + current_level_completions
```

Comment:
No need to load help if not at 'root' level.

```suggestion
    if context.current_level().name == 'root':
        return help_module.list_help_topics() + current_level_completions
```